sdk/communication/Azure.Communication.Sms/tests/SmsClientLiveTests.cs

```diff
@@ -30,15 +30,13 @@ public async Task SendingSmsMessage()
             #region Snippet:Azure_Communication_Sms_Tests_Troubleshooting
             try
             {
-                #region Snippet:Azure_Communication_Sms_Tests_SendAsync
                 SmsSendResult result = await client.SendAsync(
                    //@@ from: "+18001230000" // Phone number acquired on your Azure Communication resource
                    //@@ to: "+18005670000",
```

Comment:
can we remove the //@@ if it's no longer in a region?

Reply:
This is still within the troubleshooting snippet (line 30)